keystore/src/connection/mod.rs

```diff
@@ -107,6 +107,7 @@ pub trait DatabaseConnection<'a>: DatabaseConnectionRequirements {
 
     async fn update_key(&mut self, new_key: &DatabaseKey) -> CryptoKeystoreResult<()>;
 
+    /// Default implementation of wipe
     async fn wipe(self) -> CryptoKeystoreResult<()>;
```

Comment:
This doesn't have a default implementation. Docs would be better as i.e.


```suggestion
    /// Clear all data from the database and close it.
    async fn wipe(self) -> CryptoKeystoreResult<()>;
```